### DISCUSSION TITLE
Laravel Permission to Orchid

### DISCUSSION THREAD

**Discussion by vanvanni** [orchidsoftware/platform#1710] [Q&A]
Hey Community,

I currently use Laravel permissions but now with the implementation of Orchid it conflicts. What is the best way to solve this? They are not compatible with eachother and when I remove laravel permisions it's HasRoles trait from the users model I get this
```
App\Orchid\Presenters\UserPresenter::title(): Return value must be of type string, null returned (View: C:\laragon\www\project\vendor\orchid\platform\resources\views\auth\lockme.blade.php)
```

**Comment by maatata** [comment]
Hi

I assume currently you have something like the following in your UserPresenter `title` method:
```
return $this->entity->name;
```

You could change that to:
```
return $this->entity->name ?? '';
```

It hopefully should get you going.

I hope this helps
